base/applications/winver/winver.c

```diff
@@ -16,5 +17,7 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPWSTR lpCmdLi
     UNREFERENCED_PARAMETER(lpCmdLine);
     UNREFERENCED_PARAMETER(nCmdShow);
 
+    InitCommonControls();
```

Comment:
It'd be more appropriate to use `InitCommonControlsEx` since the function you're using is obsolete.

Reply:
Does not matter,
this function is just used to link against the dll.

Reply:
I did it, but actually InitCommonControl() was more than enough for the purpose.